compose/config/fields_schema.json

```diff
@@ -2,15 +2,18 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
 
   "type": "object",
+  "id": "fields_schema.json",
 
   "patternProperties": {
     "^[a-zA-Z0-9._-]+$": {
       "$ref": "#/definitions/service"
     }
   },
+  "additionalProperties": false,
 
   "definitions": {
     "service": {
+      "id": "#/definitions/service",
```

Comment:
If we change the path of that definition to something else we'll have to change the ID as well, which goes against the idea of what an ID is supposed to be.

Alternatives: `service_definition`, `service_description`, `service_properties`?